modules/wowchemy/assets/js/wowchemy.js

```diff
@@ -75,15 +79,25 @@ function toggleSearchDialog() {
 function fixHugoOutput() {
   // Fix Goldmark table of contents.
   // - Must be performed prior to initializing ScrollSpy.
-  $('#TableOfContents').addClass('nav flex-column');
-  $('#TableOfContents li').addClass('nav-item');
-  $('#TableOfContents li a').addClass('nav-link');
+  if (document.querySelector('#TableOfContents')) {
+      document.querySelector('#TableOfContents').classList.add('nav flex-column');
+  }
+  if (document.querySelector('#TableOfContents li')) {
+      document.querySelector('#TableOfContents li').classList.add('nav-item');
```

Comment:
There can be multiple `<li>`, so surely this requires a foreach statement?

Similar feedback applies to the other instances where the query selector matches multiple elements.

Reply:
Have you tested all these changes?

Reply:
@Konafets I see you have pushed some more changes but have not responded to the above comments yet?

Do the new commits fix all the issues and have you now tested that the PR works?

Reply:
Hi, 

yes after your question I get back and tested all changes successfully. This means, multiple tables are handled correctly, the table of content and the checkboxes get the bullets removed. From my tests, it works like it did before.